worldedit-core/src/main/java/com/sk89q/worldedit/command/GenerationCommands.java

```diff
@@ -122,67 +122,92 @@ public void cyl(Player player, LocalSession session, EditSession editSession, Pa
     }
 
     @Command(
-        aliases = { "/hsphere" },
-        usage = "<block> <radius>[,<radius>,<radius>] [raised?]",
-        desc = "Generates a hollow sphere.",
-        help =
-            "Generates a hollow sphere.\n" +
-            "By specifying 3 radii, separated by commas,\n" +
-            "you can generate an ellipsoid. The order of the ellipsoid radii\n" +
-            "is north/south, up/down, east/west.",
-        min = 2,
-        max = 3
+            aliases = { "/hemisphere" },
```

Comment:
It looks like your formatted these annotations incorrectly. I'm not sure if we have an official formatter profile for IntelliJ or Eclipse though.